tests/packed-load-64-1.ispc

```diff
@@ -0,0 +1,13 @@
+export void f_f(uniform float RET[], uniform float aFOO[]) {
+    uniform unsigned int64 a[programCount];
+    #pragma ignore warning(perf)
```

Comment:
Why do we have this pragma here, but not in other tests?

Reply:
We get warnings because of 'unsigned -> other type' conversions here

Reply:
For this, there is an 'unsigned <--> other type' conversion. So, it caused a warning.